Realm/RLMSyncUtil_Private.h

```diff
@@ -22,6 +22,17 @@
 #import <Realm/RLMRealmConfiguration.h>
 #import <Realm/RLMSyncCredentials.h>
 
+typedef NS_ENUM(NSUInteger, RLMSyncSystemErrorKind) {
```

Comment:
Since some errors with codes in this enum can be reported to users via the public API, does it make sense to make this enum public as well?